packages/ra-core/src/dataProvider/useGetManyAggregate.ts

```diff
@@ -73,6 +73,7 @@ export const useGetManyAggregate = <RecordType extends RaRecord = any>(
     const queryClient = useQueryClient();
     const queryCache = queryClient.getQueryCache();
     const { ids, meta } = params;
+    const { enabled = ids.length > 0, ...restOptions } = options;
```

Comment:
There is still a mystery here: how come the ReferenceManyFieldController calls useGetMany with an empty array?